frontend/src/templates/Header/PublicHeader.tsx

```diff
@@ -0,0 +1,79 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+import { chakra, Flex, FlexProps, HStack } from '@chakra-ui/react'
+
+import { ReactComponent as BrandHortSvg } from '~assets/svgs/brand/brand-hort-colour.svg'
+import { ReactComponent as BrandMarkSvg } from '~assets/svgs/brand/brand-mark-colour.svg'
+import Button from '~components/Button'
+import Link from '~components/Link'
+
+const BrandHortLogo = chakra(BrandHortSvg)
+const BrandSmallLogo = chakra(BrandMarkSvg)
+
+type PublicHeaderLink = {
+  label: string
+  href: string
+  showOnMobile?: boolean
+}
+
+export interface PublicHeaderProps {
+  /** Footer links to display, if provided. */
+  publicHeaderLinks?: PublicHeaderLink[]
+}
+
+export const PublicHeader = ({
+  publicHeaderLinks,
+}: PublicHeaderProps): JSX.Element => {
+  return (
+    <PublicHeader.Container>
+      <Link title="Form Logo" href="https://form.gov.sg/">
+        <BrandSmallLogo display={{ base: 'block', sm: 'none' }} w="40px" />
+        <BrandHortLogo display={{ base: 'none', sm: 'block' }} w="124.25px" />
+      </Link>
+      <HStack>
+        <HStack
+          paddingEnd={{ base: '1.5rem', md: '2rem', xl: '2.5rem' }}
+          spacing={{ base: '1.5rem', md: '2rem', xl: '2.5rem' }}
+          textStyle="subhead-1"
+        >
+          {publicHeaderLinks?.map(({ label, href, showOnMobile }, index) => (
+            <Link
+              display={{ base: showOnMobile ? 'block' : 'none', md: 'block' }}
+              w="fit-content"
+              variant="standalone"
+              color="primary.500"
+              key={index}
+              href={href}
+            >
+              {label}
+            </Link>
```

Comment:
Since we have a `showOnMobile` prop, should we not render the `Link` at all instead of setting `display` to `'none'`?

```tsx
{publicHeaderLinks?.map(({ label, href, showOnMobile }, index) => (
  showOnMobile ?  
    <Link
      // no need for display prop here anymore
      w="fit-content"
      variant="standalone"
      color="primary.500"
      key={index}
      href={href}
    >
      {label}
    </Link>
   : null)
}
```

Reply:
no need this for now as we have only one Help link that will show on mobile by default until Toolset arrives, confirmed by @pearlyong. deleting this prop until then.